src/store/PeopleContext.tsx

```
@@ -0,0 +1,59 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { Person } from '../types/Person';
+import { getPeople } from '../api';
+
+export const PeopleContext = React.createContext({
+  people: [] as Person[],
+  isLoading: false,
+  error: false,
+  loadPeople: () => {},
+});
```

Minor suggestion: consider declaring a typed interface for the context value rather than inlining the default. This makes the shape explicit and improves discoverability and TypeScript ergonomics for consumers. For example:

```ts
interface PeopleContextValue {
  people: Person[];
  isLoading: boolean;
  error: boolean;
  loadPeople: () => Promise<void>;
}

export const PeopleContext = React.createContext<PeopleContextValue | undefined>(undefined);
```

Then in a custom hook `usePeople()` you can throw if the provider is missing, helping surface integration mistakes.